src/main/routes/login.js

```diff
@@ -11,11 +12,12 @@ router.get('/', function (req, res, next) {
 })
 
 /* POST login page */
-router.post('/', (req, res, next) => {
-	// hard coded username and password
-	if (req.body.username === 'user' && req.body.password === 'password') {
+router.post('/', async (req, res, next) => {
+	try{
+		var in_database = await user_lib.get(req.body.username)
```

Comment:
So `in_database` is just a dummy variable? If the user isn't in the database `user_lib.get` throws an error?

Reply:
Yeah I have it there because the function returns the user from the database. The function will throw an error whenever the username doesn't exist or there is no username.